### ISSUE TITLE
Create a chainer operator image tagged 0.3

### ISSUE THREAD

**Issue by jlewi** [kubeflow/chainer-operator#25]
The chainer component should be using an image tagged 0.3 for the 0.3. release.

Currently its using the "latest" tag:
https://github.com/kubeflow/kubeflow/blob/master/kubeflow/chainer-job/prototypes/chainer-operator.jsonnet#L6

Ideally the image would be published to gcr.io/kubeflow-images-public as well.

**Comment by jlewi** [comment]
@everpeace Can you pick this up?

**Comment by everpeace** [comment]
Tasks

- [ ] add chainer to kubeflow releaser
- [ ] change ci for using `gcr.io/kubeflow-images-public`'s latest image
- [ ] change  kubeflow's ksonnet prototype so that it uses `v0.3.0` image.

**Comment by jlewi** [comment]
How's this coming?

This week we revamped our image auto release process. So the chainer postsubmits should now be able to build and push an image to gcr.io/kubeflow-images-public.

I added updated instructions here:
https://github.com/kubeflow/kubeflow/blob/master/docs_dev/releasing.md#image-auto-release

Unfortunately this means there is a bit of churn. i.e. the release workflow you just added in kubeflow/kubeflow#1549 should move to kubeflow/chainer and be run on postsubmit.

Sorry for that.
